content/rdi/installation/install-k8s.md

```diff
@@ -113,50 +113,79 @@ Run `redis-di status` to check the status of the installation.
 
 ## Install RDI CLI on Kubernetes Cluster
 
-### Add CLI pod
+### Add CLI deployment
 
 ```bash
-cat << EOF > /tmp/redis-di-cli-pod.yml
-apiVersion: v1
-kind: Pod
+cat << EOF > /tmp/redis-di-cli-deployment.yml
+apiVersion: apps/v1
+kind: Deployment
 metadata:
   name: redis-di-cli
-  labels:
-    app: redis-di-cli
 spec:
-  containers:
-    - name: redis-di-cli
-      image: docker.io/redislabs/redis-di-cli
-      volumeMounts:
-      - name: config-volume
-        mountPath: /app
-      - name: jobs-volume
-        mountPath: /app/jobs
-  volumes:
-    - name: config-volume
-      configMap:
-        name: redis-di-config
-        optional: true
-    - name: jobs-volume
-      configMap:
-        name: redis-di-jobs
-        optional: true
+  replicas: 1
+  selector:
+    matchLabels:
+      app: redis-di-cli
+  template:
+    metadata:
+      labels:
+        app: redis-di-cli
+    spec:
+      containers:
+        - name: redis-di-cli
+          image: docker.io/redislabs/redis-di-cli:latest
+          volumeMounts:
+          - name: config-volume
+            mountPath: /app
+          - name: jobs-volume
+            mountPath: /app/jobs
+      volumes:
+        - name: config-volume
+          configMap:
+            name: redis-di-config
+            optional: true
+        - name: jobs-volume
+          configMap:
+            name: redis-di-jobs
+            optional: true
 EOF
-kubectl apply -f /tmp/redis-di-cli-pod.yml
+kubectl apply -f /tmp/redis-di-cli-deployment.yml           
+```
+
+After creation of the deployment, the RDI CLI will be available as a pod in the cluster. It can be scaled down to 0 replicas when not in use or during maintenance. Use the following commands to scale down/up the deployment:
```

Comment:
Split it to different sections:
```
After creation of the deployment, the RDI CLI will be available as a pod in the cluster. 

It can be scaled down to 0 replicas when not in use or during maintenance. Use the following commands to scale down/up the deployment:
```

Reply:
fixed